### PR TITLE
Wait for recovery cleaner to complete in LabelScanStoreLoggingTest

### PR DESCRIPTION
**Pull request by MishaDemianenko**
Fix recently introduced component start race by waiting for cleanup job
to be always completed before checking recovery log messages.

### CODE REVIEW

**Comment by MishaDemianenko** [comment]
piu piu @burqen  :)

File: community/common/src/main/java/org/neo4j/kernel/lifecycle/Lifecycles.java [hunk 1]
```diff
@@ -27,43 +29,57 @@ private Lifecycles()
 
     public static Lifecycle multiple( final Iterable<? extends Lifecycle> lifecycles )
     {
-        return new Lifecycle()
+        return new CombinedLifecycle( lifecycles );
+    }
+
+    public static Lifecycle multiple( Lifecycle... lifecycles )
+    {
+        return new CombinedLifecycle( Arrays.asList( lifecycles ) );
```
Comment (burqen) [inline community/common/src/main/java/org/neo4j/kernel/lifecycle/Lifecycles.java]: We can reuse `multiple( Iterable lifecycles )` here.

**Review by burqen — CHANGES_REQUESTED** [review]
Small possibility for improvement, otherwise changes look awesome.

**Comment by MishaDemianenko** [comment]
@burqen updated, thx!

**Review by burqen — APPROVED** [review]

**Comment by burqen** [comment]
@MishaDemianenko PR approved. Build does not look very happy though.

**Pull request merged**